pallets/runtime/tests/all/transaction_payment_test.rs

```diff
@@ -0,0 +1,765 @@
+use polymesh_common_utilities::{
+    traits::{
+        asset::AcceptTransfer,
+        balances::{self, AccountData, CheckCdd},
+        identity::IdentityTrait,
+        CommonTrait,
+    },
+    SystematicIssuers,
+};
+use primitives::{IdentityId, Permission};
+use pallet_balances::Call as BalancesCall;
+use pallet_transaction_payment::RuntimeDispatchInfo;
+
+
+use codec::Encode;
+use frame_support::{
+    impl_outer_dispatch, impl_outer_event, impl_outer_origin, parameter_types,
+    weights::{
+        DispatchClass, DispatchInfo, GetDispatchInfo, PostDispatchInfo, Weight,
+        WeightToFeeCoefficient, WeightToFeeCoefficients, WeightToFeePolynomial,
+    },
+};
+use smallvec::smallvec;
+use sp_core::H256;
+use sp_runtime::{
+    testing::{Header, TestXt},
+    traits::{BlakeTwo256, IdentityLookup},
+    Perbill,
+};
+use std::cell::RefCell;
+
+pub type AccountId = u64;
+pub type Balance = u128;
+
+const CALL: &<Runtime as frame_system::Trait>::Call =
+&Call::Balances(BalancesCall::transfer(2, 69));
+
+impl_outer_dispatch! {
+    pub enum Call for Runtime where origin: Origin {
+        pallet_balances::Balances,
+        frame_system::System,
+    }
+}
+
+impl_outer_event! {
+    pub enum Event for Runtime {
+        system<T>,
+        pallet_balances<T>,
+    }
+}
+
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub struct Runtime;
```

Comment:
Can the runtime struct be `super::storage::testStorage` instead?

Reply:
I've just move it to avoid circle dep, and I agree that we should use here the `TestStorage`, so I am going to a task for that.